pytorch_lightning/profiler/base.py

```diff
@@ -26,7 +26,14 @@
 
 
 class AbstractProfiler(ABC):
-    """Specification of a profiler."""
+    """Specification of a profiler.
+
+    See deprecation warning below
+
+    .. deprecated:: v1.6
+        `AbstractProfiler` was deprecated in v1.6 and will be removed in v1.8.
+        Please use `BaseProfiler` instead
```

Comment:
It might be a bit silly because this doesn't raise a warning.

But it would be good to add a "fake" deprecation test so we remember this got deprecated in the future.